
This hack seems to fix too "slow" hdmi audio playback...


diff --speed-large-files --no-dereference --minimal -Naur linux-6.14-rc1/drivers/clk/sunxi-ng/ccu-sun50i-h616.c linux-6.14-rc1/drivers/clk/sunxi-ng/ccu-sun50i-h616.c
--- linux-6.14-rc1/drivers/clk/sunxi-ng/ccu-sun50i-h616.c	2025-02-03 00:39:26.000000000 +0100
+++ linux-6.14-rc1/drivers/clk/sunxi-ng/ccu-sun50i-h616.c	2025-02-09 16:55:39.896686428 +0100
@@ -235,10 +229,8 @@
 	.m		= _SUNXI_CCU_DIV(16, 6),
 	.sdm		= _SUNXI_CCU_SDM(pll_audio_sdm_table,
 					 BIT(24), 0x178, BIT(31)),
-	.fixed_post_div = 2,
 	.common		= {
-		.features	= CCU_FEATURE_FIXED_POSTDIV |
-				  CCU_FEATURE_SIGMA_DELTA_MOD,
+		.features	= CCU_FEATURE_SIGMA_DELTA_MOD,
 		.reg		= 0x078,
 		.hw.init	= CLK_HW_INIT("pll-audio-hs", "osc24M",
 					      &ccu_nm_ops,
